src/pages/Explorer/Category/UserCreation.svelte

```diff
@@ -21,6 +24,10 @@
         class="$style.actions"
         isOwner={$currentUser && user && user.id === $currentUser.id}
         {url}
+        {item}
+        {type}
+        showCommentAction={commentsCount >= 0}
```

Comment:
`commentsCount` can't be negative, hence this expression always results in `true`
```suggestion
        showCommentAction
```

Reply:
It can be null

Reply:
alerts don't have comments and that field is null

Reply:
Makes sense 👍